libs/SalesforceNetwork/SalesforceNetwork/SalesforceNetwork/Classes/Network/Action/CSFSalesforceAction.m

```diff
@@ -197,12 +197,19 @@ - (id)contentFromData:(NSData*)data fromResponse:(NSHTTPURLResponse*)response er
                 network.securityToken = securityToken;
             }
         }
+    } else if (!content && !responseError && self.statusCodeToRaiseErrorWithEmptyResponse.count > 0) {
```

Comment:
This seems like it's working around the existing design pattern of having an inherited class that overrides `contentFromData:fromResponse:error:`, if you need custom error handling behavior.